### PR TITLE
update xc integration docs

### PR DESCRIPTION
**Pull request by eshaben**
### Description/Original PRs

The xc integration page is not new, but has undergone a big transformation.

This PR includes updates from changes made in the following PRs:

- https://github.com/PureStake/moonbeam-docs/pull/590 
- https://github.com/PureStake/moonbeam-docs/pull/547

This also includes some changes that were not translated (because I had already sent the file to crypto discover) that still need to be translated:

- https://github.com/PureStake/moonbeam-docs/pull/614/files
- https://github.com/PureStake/moonbeam-docs/pull/582/files

### CODE REVIEW

File: builders/interoperability/xcm/xc-integration.md [hunk 1]
```diff
@@ -342,79 +369,519 @@ Sovereign Account Address on Moonbase Alpha: 0x7369626ce803000000000000000000000
...
+            {
+              'PalletInstance': 3
+            }
+          ]
+        }
+      }
+    }
     ```
+
+### Register Local XC-20s (ERC-20s) {: #register-erc20s }
```
Comment (eshaben) [inline builders/interoperability/xcm/xc-integration.md:444]: this whole section needs to be translated still please!

File: builders/interoperability/xcm/xc-integration.md [hunk 2]
```diff
@@ -106,203 +89,93 @@ Sovereign Account Address on other Parachains (Generic): 0x7369626ce803000000000
...
 
-1. 在**submit the following extrinsic**下拉菜单中选择**hrmp**
-2. 选择**hrmpAcceptOpenChannel** extrinsic
-3. 输入Moonbeam平行链ID作为**sender（发送方）**
-    === "Moonbeam"
-        ```
-        2004
-        ```
+      - **Moonriver** - 提案需在[OpenGov](/learn/features/governance/#opengov){target=_blank}的General Admin Track完成，其中决定期约为{{ networks.moonriver.governance.tracks.general_admin.decision_period.time }}，执行期至少为{{ networks.moonriver.governance.tracks.general_admin.min_enactment_period.time }}
+      - **Moonbeam** - 约为{{ networks.moonbeam.democracy.vote_period.days }}天的投票期 + {{ networks.moonbeam.democracy.enact_period.days }天的执行期
```
Comment (themacexpert) [inline builders/interoperability/xcm/xc-integration.md:120]:
```suggestion
      - **Moonbeam** - 约为{{ networks.moonbeam.democracy.vote_period.days }}天的投票期 + {{ networks.moonbeam.democracy.enact_period.days }}天的执行期
```

File: builders/interoperability/xcm/xc-integration.md [hunk 3]
```diff
@@ -342,79 +219,405 @@ Sovereign Account Address on Moonbase Alpha: 0x7369626ce803000000000000000000000
...
+
+如果您计划将多个跨链资产注册到Moonbeam，则可以对多个资产重复此过程。
+
+### 计算每秒的单位 {: #calculating-units-per-second }
 
-资产注册后，团队需确认。另外，团队将提供资产ID，[XC-20预编译](/builders/interoperability/xcm/xc20/overview/#the-erc20-interface){target=_blank}地址，并设置`UnitsPerSecond`，即每秒执行XCM消息所需的Token数量。在注册时，XCM转账的目标成本为`$0.02`。`UnitsPerSecond`可能会随着Token价格的波动而更新。
+`UnitsPerSecond`是执行XCM消息时每秒收取的Token数。XCM转账的目标成本在注册时为`$0.02`。随着Token价格的波动，`UnitsPerSecond`可能会通过治理进行更新。
 
-您的XC-20预编译地址是通过将资产ID十进制数字转换为十六进制来计算的，并在前面加上F，直到您获得加上“0x"后达到40个字符长度的地址。关于如何计算XC-20预编译，请参考外部XC-20教程的[计算外部XC-20预编译地址](/builders/interoperability/xcm/xc20/xc20/#calculate-xc20-address){target=_blank}的部分。
+计算资产的`UnitsPerSecond`最简单方法是通过[xcm-tools](https://github.com/PureStake/xcm-tools){target=_blank}的[`calculate-units-per-second.ts`脚本](https://github.com/PureStake/xcm-tools/blob/main/scripts/calculate- units-per-second.ts){target=_blank}。要运行脚本，您必须提供以下内容：
```
Comment (themacexpert) [inline builders/interoperability/xcm/xc-integration.md:510]:
```suggestion
计算资产的`UnitsPerSecond`最简单方法是通过[xcm-tools](https://github.com/PureStake/xcm-tools){target=_blank}的[`calculate-units-per-second.ts`脚本](https://github.com/PureStake/xcm-tools/blob/main/scripts/calculate-units-per-second.ts){target=_blank}。要运行脚本，您必须提供以下内容：
```

File: builders/interoperability/xcm/xc-integration.md [hunk 4]
```diff
@@ -342,79 +219,405 @@ Sovereign Account Address on Moonbase Alpha: 0x7369626ce803000000000000000000000
...
+
+您可以使用Polkadot.js Apps为HRMP操作生成调用数据，但是[xcm-tools GitHub repository](https://github.com/PureStake/xcm-tools){target=_blank}可以为您构建，这是该过程的推荐工具。其应该适用于任何包含 [Polkadot XCM Pallet](https://github.com/paritytech/polkadot/tree/master/xcm/pallet-xcm){target=_blank}的链，它们会先通过[XCM-Transactor Pallet](https://github.com/PureStake/moonbeam/tree/master/pallets/xcm-transactor){target=_blank}操作。
+
+```
+git clone https://github.com/PureStake/xcm-tools && \
+cd xcm-tools && \
+yarn
+```
+
+[xcm-tools repository](https://github.com/PureStake/xcm-tools){target=_blank}有一个用于HRMP交互的特定脚本，称为[`hrmp-channel-manipulator.ts`](https:/ /github.com/PureStake/xcm-tools/blob/main/scripts/hrmp-channel-manipulator.ts){target=_blank}。此命令为特定HRMP操作生成编码的调用数据，只需提供正确的详细信息即可。该脚本使用DepositAsset XCM指令构建XCM消息，但不是RefundSurplus。
```
Comment (themacexpert) [inline builders/interoperability/xcm/xc-integration.md:346]:
```suggestion
[xcm-tools repository](https://github.com/PureStake/xcm-tools){target=_blank}有一个用于HRMP交互的特定脚本，称为[`hrmp-channel-manipulator.ts`](https://github.com/PureStake/xcm-tools/blob/main/scripts/hrmp-channel-manipulator.ts){target=_blank}。此命令为特定HRMP操作生成编码的调用数据，只需提供正确的详细信息即可。该脚本使用DepositAsset XCM指令构建XCM消息，但不是RefundSurplus。
```

**Review by themacexpert — APPROVED** [review]
LGTM 👍

**Pull request merged**